notebooks/rnn/lyft-rnn-lstm.py

```diff
@@ -250,8 +250,9 @@ def normalizeAgents(agents):
     dataForNormalization = []
     pb = ProgressBar(len(agents))
     pb.start()
-    for agent in agents:
-        pb.check(0, True)
```

Comment:
I suspect this fixes some freezing since instead of printing for each loop we are using the underlying chunked updated of the PB class